docs/getting-started.md

```diff
@@ -14,33 +21,24 @@ yarn d2-utils-cypress
 npx --no-install d2-utils-cypress
 ```
 
-We are going to use `d2-utils-cypress` in this document, so prefix the
-commands with your preference.
+## Install the cypress commands
 
-# Setup Cypress for your project
-
-The CLI module provides an `install` command, that will set up some
-prerequisites for Cypress, and guide you through generating a
-configuration and a local environment configuration file.
+When using the cli tool's `install` command, this will be handled for you. If
```

Comment:
I think it'd be good to also provide a copy-paste command here.

Reply:
Actually, I think this could be seen as duplicate info. What you have written in `docs/guides/setting-up-cli-tool.md` is more complete.

Reply:
I've linked to the guide from this document, that should resolve this issue?